server/integration_tests/test_data/e2e_edge_cases2/howdoesschoolsizeofurbanschoolscomparetoruralschoolsinus/chart_config.json

```diff
@@ -10,144 +10,71 @@
                 "tiles": [
                   {
                     "statVarKey": [
-                      "Count_Person_EnrolledInKindergarten"
-                    ],
-                    "title": "Population: Enrolled in Kindergarten in United States",
-                    "type": "LINE"
-                  }
-                ]
-              },
-              {
-                "tiles": [
-                  {
-                    "description": "Population: Enrolled in Kindergarten in United States",
-                    "statVarKey": [
-                      "Count_Person_EnrolledInKindergarten"
-                    ],
-                    "title": "Population: Enrolled in Kindergarten in United States",
-                    "type": "HIGHLIGHT"
-                  }
-                ]
-              }
-            ],
-            "denom": "Count_Person",
-            "title": "Population: Enrolled in Kindergarten"
-          },
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
+                      "Count_Person_EnrolledInKindergarten",
```

Comment:
This query was an edge case one that returns low-confidence matches.  It now gets fulfilled as multi-sv thing (which is what it is).